reckless_lib/src/utils.rs

```diff
@@ -11,7 +11,7 @@ pub fn create_dir_in_home(relative_path: &str) {
         .into_os_string()
         .into_string()
         .unwrap();
-    path = path + "/" + relative_path;
+    path = path + "/plugins/" + relative_path;
```

Comment:
```suggestion
    path = path + "/plugins/" + relative_path;
```

Whi this additional name?

Maybe it is better `repositories` and inside it you will go to clone all the repositories?